trackintel/io/file.py

```diff
@@ -85,18 +85,18 @@ def read_positionfixes_csv(*args, columns=None, tz=None, index_col=object(), crs
 
     df = pd.read_csv(*args, **kwargs)
     df = df.rename(columns=columns)
-    df['geom'] = list(zip(df.longitude, df.latitude))
+    
+    # construct geom column from lon and lat
+    df['geom'] = list(zip(df['longitude'], df['latitude']))
     df['geom'] = df['geom'].apply(Point)
-    df['tracked_at'] = df['tracked_at'].apply(dateutil.parser.parse)
+    
+    # transform to datatime
+    df["tracked_at"] = pd.to_datetime(df["tracked_at"])
 
-    # check and/or set timezone
+    # set timezone if none is recognized
     for col in ['tracked_at']:
         if not pd.api.types.is_datetime64tz_dtype(df[col]):
             df[col] = localize_timestamp(dt_series=df[col], pytz_tzinfo=tz, col_name=col)
-        else:
-            # dateutil parser timezones are sometimes not compatible with pandas (e.g., in asserts)
-            tz = df[col].iloc[0].tzinfo.tzname(df[col].iloc[0])
```

Comment:
Hm... I don't really remember why I put this there. There was a problem that `dateutil` and `pytz` timezones are sometimes not compatible. This line essentially transforms `dateutil` into `pytz` timezones or something like that. But maybe the error is not coming up because we are now using better asserts (e.g., is_datetime64tz_dtype)?